VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts

```diff
@@ -112,46 +112,26 @@ export class PrivateConsultationParticipantsComponent extends WRParticipantStatu
     }
 
     getPrivateConsultationParticipants(): ParticipantListItem[] {
-        const regularParticipants = this.participantsInConsultation
-            .filter(
-                c =>
-                    c.hearing_role !== HearingRole.WITNESS &&
-                    c.hearing_role !== HearingRole.OBSERVER &&
-                    c.role !== Role.QuickLinkObserver &&
-                    c.role !== Role.QuickLinkParticipant
-            )
-            .filter(c => c.hearing_role !== HearingRole.INTERPRETER)
-            .filter(
-                c =>
-                    c.hearing_role !== HearingRole.JUDGE &&
-                    c.hearing_role !== HearingRole.PANEL_MEMBER &&
-                    c.hearing_role !== HearingRole.WINGER
-            )
-            .map(c => {
-                return this.mapResponseToListItem(c);
-            });
-        const quickLinkParticipants = this.sortAndMapToListItem(
-            this.participantsInConsultation.filter(p => p.role === Role.QuickLinkParticipant)
-        );
-        return [...regularParticipants, ...quickLinkParticipants];
+        return this.sortAndMapToListItem(this.nonJudgeParticipants.filter(x => x.hearing_role !== HearingRole.WITNESS));
```

Comment:
I simplified the logic in here to use lists that we already filter as part of the base directive. This should also reduce inconsistencies.